CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/utils/DataAccessPreferenceUtils.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package au.gov.asd.tac.constellation.views.dataaccess.state;
+package au.gov.asd.tac.constellation.views.dataaccess.utils;
```

Comment:
Can you call this `utilities` for consistency with the class names convention we have going on (see my next comment).